build.gradle.kts

```diff
@@ -365,10 +373,17 @@ tasks.register("startRedis") {
       "redis-server",
       "--loglevel debug"
     )
-    exec {
-      workingDir(project.rootProject.rootDir.toPath())
+    execOperations.exec {
+      workingDir(rootDir.get().asFile.toPath())
```

Comment:
nit: I think `workingDir()` takes `File` as well, so you could probably drop the `toPath()`